src/isolate-duplicates/index.js

```diff
@@ -1,3 +1,49 @@
-function isolateDuplicates(text) {}
+function isolateDuplicates(text) {
+  if((typeof text ) != "string"){
+        throw "Please enter a valid string";
```

Comment:
Prefer to `throw Error(...)` rather than strings. Helps preserve the stack trace for debugging.